src/express.ts

```diff
@@ -306,8 +308,15 @@ export class InngestCommHandler {
 
     let res: globalThis.Response;
 
+    // Whenever we register, we check to see if the dev server is up.  This
+    // is a noop and returns false in production.
+    let registerURL = this.inngestRegisterUrl;
+    if (await available()) {
```

Comment:
Nit - I find `await available()` an odd statement to read - could we rename the `available` import to `isDevServerAvailable()` or something more specific?